zwave_js_server/model/node.py

```diff
@@ -60,6 +60,10 @@ def __init__(self, client: "Client", data: NodeDataType) -> None:
             get_value_id(self, val): Value(self, val) for val in data["values"]
         }
 
+    def __repr__(self) -> str:
+        """Return the representation."""
+        return f"{type(self).__name__}(client={self.client}, node_id={self.node_id})"
```

Comment:
I think including `client` would make it too verbose. I would prefer `<Node(id)>`